packages/react/src/patterns/blocks/LeadSpaceBlock/LeadSpaceBlock.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import {VideoPlayer} from "../../../components/VideoPlayer";
+import { LinkList } from '../../sub-patterns/LinkList';
+import { Button} from 'carbon-components-react';
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import {ImageWithCaption} from "../../../components/ImageWithCaption";
+
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+const _renderMedia = (type, data) => {
+  if (data) {
+    return (
+      <div data-autoid={`${stablePrefix}--leadspace-block__media`}
+           className={`${prefix}--leadspace-block__media`}>
+        {type === 'image' && <ImageWithCaption {...data} />}
+        {type === 'video' && <VideoPlayer {...data} />}
+      </div>
+    );
+  }
+};
+
+const LeadSpaceBlock = ({copy, title, heading, mediaType, mediaData, items, button }) => {
+
+  const pageTitle = (
```

Comment:
Make sure you're adding JSDocs.

Reply:
JSDocs for text constant? could you please mention how can I use it.

Reply:
@greeshmareddyg actually all of our components are stateless, you can see an example of one of them documented here: 

https://github.com/carbon-design-system/ibm-dotcom-library/blob/de1c2d8a9508cb32d6702bacf758058c4ea6d619/packages/react/src/patterns/sections/LeadSpace/LeadSpace.js#L88

Reply:
added the props documentation there.